### PR TITLE
add RN tintColor support

### PR DESCRIPTION
**Pull request by lucienbl**
This adds RN image `tintColor` support. 
Usage : `tint-{color}`, e.g. `tint-red-200`

### CODE REVIEW

File: src/__tests__/colors.spec.ts [hunk 1]
```diff
@@ -48,6 +48,10 @@ describe(`colors`, () => {
     expect(tw`text-black`).toEqual({ color: `#000` });
   });
 
+  test(`tint colors`, () => {
+    expect(tw`tint-black`).toEqual({ tintColor: `#000` });
```
Comment (jaredh159) [inline src/__tests__/colors.spec.ts:52]: can you add a couple more tests here, for stuff like:

- arbitrary values: `tint-[#eaeaea]`
- customized theme colors
- opacity: `tint-black/50`

they should all immediately pass, i would just feel better with the test coverage

**Comment by jaredh159** [comment]
that was quick!

thanks for the PR.  just beef up the tests as noted in the comments, and then run `npm run lint:fix` and `npm run format` to fix the CI issues, and i'll merge. 👍

**Comment by lucienbl** [comment]
I just added a few more tests, let me know if it's okay like this :)

**Comment by jaredh159** [comment]
looks great, thanks!  will release `v3.2.0` in just a sec...

**Pull request merged**